mm/2s2h/Rando/ActorBehavior/ObjGrass.cpp

```diff
@@ -604,7 +604,7 @@ std::map<s16, std::map<std::pair<float, float>, RandoCheckId>> grottoGrassMap =
           { { 2565.00f, 988.00f }, RC_GREAT_BAY_COAST_COW_GROTTO_GRASS_71 },
           { { 2595.00f, 1025.00f }, RC_GREAT_BAY_COAST_COW_GROTTO_GRASS_72 },
       } },
-    { 0,
+    { 5376,
```

Comment:
Same thing here, try to use the entrance builder for these values (just showing this one example, but please do the rest below)
```suggestion
    { ENTRANCE(GROTTOS, 16),
```